src/lib/cli.ts

```diff
@@ -25,10 +25,10 @@ import { loadJSONFile } from './utils/misc';
 import * as logger from './utils/logging';
 import { cutString } from './utils/misc';
 import { options } from './ui/options';
+import { initSonarrc } from './ui/generator';
```

Comment:
It might be only me. We have both a `rule-generator` and a `generator`,  it seems a little confusing. Maybe rename the latter one as `sonarrc-generator`.